frontend/talentsearch/src/js/components/search/SearchPools.tsx

```diff
@@ -32,8 +32,11 @@ const SearchPools: React.FunctionComponent<SearchPoolsProps> = ({
     pool.classifications as SimpleClassification[];
 
   return (
-    <div data-h2-padding="base(x1)">
-      <p data-h2-font-weight="base(700)">
+    <article
+      data-h2-padding="base(x1)"
+      aria-labelledby={`search_pool_${pool.id}`}
+    >
+      <p data-h2-font-weight="base(700)" id={`search_pool_${pool.id}`}>
```

Comment:
a doozy to wrap your head around
a question or two, is this for accessibility purposes and not testing purposes? enabling linking to a card?

Reply:
is there any point where test run time is considered a problem?

![image](https://user-images.githubusercontent.com/40485260/205776142-3ff7fb21-6249-4950-ab29-e3b2601c6606.png)

Reply:
> is this for accessibility purposes and not testing purposes? enabling linking to a card

Yeah, it's for testing purposes.  It's probably good for a11y too but it's mostly there to help Cypress find the right card.

Reply:
> is there any point where test run time is considered a problem?

Certainly there has to be a line somewhere.  This test could be broken up to test each filter individually but in aggregate it would take much longer since the test would have to be set up over and over instead of just once like it is now.  I'd suggest that the time is well spent on this test since this page has been a rich source of bugs so far.